.github/workflows/ansible-lint.yml

```diff
@@ -0,0 +1,27 @@
+name: Ansible Lint
+
+on:
+  workflow_call:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Ansible Lint
+        # Uses the slim image to speed up build time
+        uses: github/super-linter/slim@v4
+        env:
+          # Only parse new or edited files
+          VALIDATE_ALL_CODEBASE: false
+          # Just validate ansible related files
+          VALIDATE_ANSIBLE: true
+          # Check every file starting at the root directory
+          ANSIBLE_DIRECTORY: .
```

Comment:
This just checks everything under ".github".

Reply:
`ANSIBLE_DIRECTORY` is relative to `DEFAULT_WORKSPACE`. I added `DEFAULT_WORKSPACE: /` to make sure that all ansbile related files starting at root directory will be linted.